lib/record_store/cli.rb

```diff
@@ -169,7 +171,7 @@ def secrets
       end
 
       secrets = %x(ejson decrypt #{RecordStore.secrets_path.sub(/\.json\z/, ".#{environment}.ejson")})
-      if $?.exitstatus == 0
+      if $CHILD_STATUS.exitstatus == 0
```

Comment:
Maybe use

```suggestion
      if $CHILD_STATUS.success?
```

It should be the same here